cms/static/js/views/video_thumbnail.js

```diff
@@ -174,10 +223,23 @@ define(
                 this.readMessages([gettext('Video image upload completed')]);
             },
 
-            imageUploadFailed: function() {
+            imageUploadFailed: function(event, data) {
+                var errorText = JSON.parse(data.jqXHR.responseText)['error'];
                 this.action = 'error';
                 this.setActionInfo(this.action, true);
-                this.readMessages([gettext('Video image upload failed')]);
+                this.showErrorMessage(errorText);
+            },
+
+            showErrorMessage: function(errorText) {
+                this.readMessages([gettext('Video image upload failed'), errorText]);
+                HtmlUtils.setHtml(
```

Comment:
@muhammad-ammar There is a bug here, after testing many times found that when an error has been shown, we can't upload image again. After debugging found out that we are updating HTML, so view has changed so event is not caught by file upload handler. I think,  this would mean that I won't be able to tag doc team on sandbox.